#! /bin/sh /usr/share/dpatch/dpatch-run
## 10_slabtop.dpatch by  <csmall@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Cleanup slabtop man page #282168

@DPATCH@
diff -urNad procps-3.2.7~/slabtop.1 procps-3.2.7/slabtop.1
--- procps-3.2.7~/slabtop.1	2006-06-18 15:10:24.000000000 +1000
+++ procps-3.2.7/slabtop.1	2006-06-26 12:31:42.000000000 +1000
@@ -7,36 +7,39 @@
 slabtop \- display kernel slab cache information in real time
 
 .SH SYNOPSIS
-.BI "slabtop [ " options " ] "
+.B slabtop
+.RI [ options ]
 
 .SH DESCRIPTION
-.BR slabtop (1)
+.B slabtop
 displays detailed kernel slab cache information in real time.  It displays a
 listing of the top caches sorted by one of the listed sort criteria.  It also
 displays a statistics header filled with slab layer information.
 
 .SH OPTIONS
 Normal invocation of 
-.BR slabtop (1)
+.B slabtop
 does not require any options.  The behavior, however, can be fine-tuned by
 specifying one or more of the following flags:
 .TP
-.B \-\^\-delay=n, \-d n
-Refresh the display every n seconds.  By default,
-.BR slabtop (1)
+.B \-\-delay=\fIn\fR, \fB\-d \fIn
+Refresh the display every 
+.I n
+in seconds.  By default,
+.B slabtop
 refreshes the display every three seconds.  To exit the program, hit
 .BR q.
 .TP
-.B \-\^\-sort=S, \-s S
-Sort by S, where S is one of the sort criteria.
+.B \-\-sort=\fIS\fR, \fB\-s\fR \fIS
+Sort by \fIS\fR, where \fIS\fR is one of the sort criteria.
 .TP
-.B \-\^\-once, \-o
+.B \-\-once\fR, \fB\-o
 Display the output once and then exit.
 .TP
-.B \-\^\-version, \-V
+.B \-\-version\fR, \fB\-V
 Display version information and exit.
 .TP
-.B \-\^\-help
+.B \-\-help
 Display usage information and exit.
 
 .SH SORT CRITERIA
@@ -78,7 +81,7 @@
 sort by cache utilization
 
 .SH COMMANDS
-.BR slabtop (1)
+.B slabtop
 accepts keyboard commands from the user during use.  The following are
 supported.  In the case of letters, both cases are accepted.
 
@@ -94,7 +97,9 @@
 Quit the program.
 
 .SH FILES
-.IR /proc/slabinfo " \-\- slab information"
+.TP
+.I /proc/slabinfo 
+slab information
 
 .SH "SEE ALSO"
 .BR free (1),
@@ -104,7 +109,7 @@
 
 .SH NOTES
 Currently,
-.BR slabtop (1)
+.B slabtop
 requires a 2.4 or later kernel (specifically, a version 1.1 or later
 .IR /proc/slabinfo ).
 Kernel 2.2 should be supported in the future.
@@ -116,7 +121,7 @@
 .SH AUTHORS
 Written by Chris Rivera and Robert Love.
 
-.BR slabtop (1)
+.B slabtop
 was inspired by Martin Bligh's perl script,
 .BR vmtop .
 The procps package is maintained by Albert Cahalan <albert@users.sf.net>.
